assets/styles/main.scss

```diff
@@ -450,7 +450,24 @@ $logo-width: 130px;
 }
 
 .footer-logos {
-  padding: 2rem 0 2rem;
+  padding-top: 1rem;
+}
+
+.footer-content {
+  padding: 0 2rem 0 2rem;
+}
```

Comment:
I don't think this is necessary since the container provides spacing on left and right at smaller breakpoints. This actually makes the logos not align with the other content.